tests/e2e/run-scanner-v4.sh

```diff
@@ -16,20 +16,62 @@ source "$ROOT/tests/scripts/setup-certs.sh"
 
 set -euo pipefail
 
+# Could also use
+#   CHART_BASE="/rhacs"
+#   export DEFAULT_IMAGE_REGISTRY="quay.io/rhacs-eng"
+# for the opensource flavour.
```

Comment:
```suggestion
# for the RHACS flavour.
```

nit: I think you meant the RHACS flavour?

Reply:
I think Moritz meant "flavor" 😄

Reply:
changed

Reply:
@vladbologa, can we resolve this?